data/campaigns/Northern_Rebirth/scenarios/05a_01_The_Pursuit.cfg

```diff
@@ -2207,10 +2319,38 @@
             x,y=35,15
         [/filter]
 
-        [message]
-            speaker=unit
-            message= _ "Gee, what’s with these bodies floating around? Is this river some sort of body disposal?"
-        [/message]
+        [if]
+            [variable]
+                name=unit.id
+                equals="Abhai"
+            [/variable]
+            [then]
+                [message]
+                    speaker=Abhai
+                    message= _ "More bodies. I am unsure what to make of this."
+                [/message]
+            [/then]
+            [else]
+                [if]
+                    [variable]
+                        name=unit.id
+                        equals="Krash"
+                    [/variable]
+                    [then]
+                        [message]
+                            speaker="Krash"
+                            message= _ "More stink."
+                        [/message]
+                    [/then]
+                    [else]
+                        [message]
+                            speaker=unit
+                            message= _ "Gee, what’s with these bodies floating around? Is this river some sort of body disposal?"
```

Comment:
There's a meaning here that's lost in Abhai and Krash's lines - that the corpses are drifting down from Malifor's throneroom.

Reply:
Makes sense. I'll update them. I thought the bodies were because of the nearby nagas, since I figured Malifor would have used the bodies for undead.

Reply:
I'm not sure where the meaning is lost - it looks the same as what was in the original text. Unless something got changed in an earlier commit and then reverted?

Reply:
Ah, I see. I read the original (generic) remark as speaking of the corpses in the river in general. While Abhai and Krash only refer to a singular corpse, the hex at 35,15, visually, has just the one skeleton.

But the original author did agree with your assessment, so I'll try to update with something suitable.